src/Compatibility/Core/src/WinUI/AccessibilityExtensions.cs

```diff
@@ -6,91 +6,6 @@ namespace Microsoft.Maui.Controls.Compatibility.Platform.UWP
 {
 	public static class AccessibilityExtensions
 	{
-		public static void SetAutomationPropertiesAutomationId(this FrameworkElement Control, string id)
```

Comment:
Yes this all moved into Controls.Core

I just left "SetBackButtonTitle" because according to the comment it seems like it doesn't work at all so I figured once we work on the backbutton in maui we can fix and move that method back to controls.core